src/Serilog/Serilog.csproj

```diff
@@ -24,16 +24,22 @@
     <PackageReference Include="System.ValueTuple" Version="4.5.0" />
   </ItemGroup>
 
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'net47' ">
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+  </PropertyGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net47' " />
+
   <PropertyGroup Condition=" '$(TargetFramework)' == 'net471' ">
     <DefineConstants>$(DefineConstants);FEATURE_ITUPLE</DefineConstants>
     <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
   </PropertyGroup>
+
   <ItemGroup Condition=" '$(TargetFramework)' == 'net471' ">
     <PackageReference Include="System.ValueTuple" Version="4.5.0" />
```

Comment:
I don't think this is actually needed for `net471`? (I notice it's missing from the added `net47` so seems like we should get rid of it from both).

Reply:
I'll commit an edit so that we can finalize this one :-)